### PR TITLE
html structure for personal website

### PR DESCRIPTION
**Pull request by johdyn**
* This is the first version of the website
* It includes basic html structure, some content and some images

### CODE REVIEW

**Comment by hugh-burgess** [comment]
LGTM

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <header>
+      <h1>Johan Dynnesen</h1>
+      <nav>
+        <ul>
+          <li><a href="#contact">Contact</a></li>
+          <li><a href="#about">About</a></li>
+        </ul>
+      </nav>
+      <img
+        width="400"
```
Comment (namirsab) [inline index.html:17]: For now this is fine as you don't have any other way to style the images, but keep in mind this is not meant to be used for styling.

You can read more information here: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML#width_and_height

**Review by namirsab — APPROVED** [review]
Looks good to me! 

Just as recommendation, i'd add more content so tomorrow you have more to style. Please check the "wireframes" we posted in the slack channel for inspiration: https://neuefische-students.slack.com/archives/G01Q1DL5F41/p1614864290074900

File: index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <a href=""
+          ><img width="50" height="50" src="images/Codepen_Logo.png"
+        /></a>
+      </section>
+      <section>
+        <h2 id="about">About</h2>
+        <p>
+          Johan is a Danish drummer and composer, living in
+          <strong>Cologne, Germany.</strong> <br />He is currently studying web
+          development with Neue Fische.
```
Comment (mpagels) [inline index.html:56]: Found it always strange to read something in third person on a personal website  ;-) But this also a personal thing. So nothing to see here. :-P

**Review by mpagels — APPROVED** [review]
Very good job Johan.

**Comment by soerendolling** [comment]
lgtm

**Review by Alexander-Schulte — COMMENTED** [review]
LGTM, good job! 🚀

**Pull request merged**